src/components/AutoExpandingTextArea.tsx

```diff
@@ -26,6 +26,7 @@ const AutoExpandingTextArea: FC<Props> = (props) => {
       <Textarea
         {...(textAreaProps as TextareaProps)}
         rows={dynamicHeight ? undefined : rows}
+        className="text-area"
```

Comment:
I think you forgot to update it here :)
```suggestion
        className="auto-expanding-textarea"
```

Reply:
oops, I think I'm definitely a little tired :disappointed: